.github/CODEOWNERS

```
@@ -51,10 +52,9 @@ packages/web-components @chrisdholt @EisenbergEffect @nicholasrice
 # component-demo/
 public-docsite/ @ecraig12345
 public-docsite-resources/ @ecraig12345 @justSlone
-perf-test/ @kubkon
+perf-test/ @ecraig12345 @jdhuntington
```

I know almost nothing about this, so it would be better to list the whole build team as owners.
```suggestion
perf-test/ @microsoft/fluentui-react-build
```